### PR TITLE
Updated log in flow and Parse User object.

### PR DESCRIPTION
**Pull request by hashamali**
## Reviewers
- [x] @dbuscaglia 
- [ ] @jcomo

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -28,6 +28,7 @@
 
         <activity
             android:name=".actvitiy.LoginActivity"
+            android:theme="@style/AppTheme.NoActionBar"
             android:label="@string/app_name" >
```
Comment (dbuscaglia) [inline app/src/main/AndroidManifest.xml:32]: What is this from?

Comment (hashamali) [inline app/src/main/AndroidManifest.xml:32]: thangs

File: app/src/main/java/it/quip/android/model/User.java [hunk 2]
```diff
@@ -93,7 +125,38 @@ public static User fromJSON(JSONObject userJson) {
...
+            currentUser.setFacebookId(facebookId);
+            currentUser.saveInBackground();
+        }
+
+        return currentUser;
+    }
+
+    private List<Long> circlesToIds() {
+        List<Long> circleIds = new ArrayList<Long>();
+        for (Circle circle : getCircles()) {
```
Comment (dbuscaglia) [inline app/src/main/java/it/quip/android/model/User.java:155]: Lets not forget to refactor UUIDs to parse object id.

File: app/src/main/res/values/colors.xml [hunk 3]
```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <color name="quipit_main">#FD8917</color>
+    <color name="quipit_alt">#FFFFFF</color>
```
Comment (dbuscaglia) [inline app/src/main/res/values/colors.xml:4]: Can we not use FFFFFFs here and just use android.colors.white

**Comment by dbuscaglia** [comment]
Minor thangs, but otherwise LGTM

**Comment by dbuscaglia** [comment]
More of a TODO, but don't forget to implement the Parcel param constructor to use setters

**Pull request merged**